project/src/main/webapp/event-edit.js

```diff
@@ -20,4 +20,7 @@ async function getEventDetails() {
     `${data['date'].month}/${data['date'].dayOfMonth}/${data['date'].year}`;
   document.getElementById('time').value = data['time'];
   document.getElementById('location').value = data['location'];
+  const eventEditForm = document.getElementById('create-event');
+  eventEditForm.action =
+      `/create-event?eventId=${eventId}`;
```

Comment:
this will double POST -- the html says '  <form id="create-event" action="/create-event" method="POST">' on submit

Reply:
this line will change the action rather than appending to action, seems fine